### PR TITLE
Translate welcome and variables

### PR DESCRIPTION
**Pull request by rschio**
Translate the Welcome and Variables sections to pt-BR.

### CODE REVIEW

**Comment by ardan-bkennedy** [comment]
Hi Rschio, we have three people working on this. Can you send me an email to bill@ardanlabs.com and I will connect you with that group.

**Comment by rschio** [comment]
Hi Bill, sure!

File: _content/tour/por/variables/answer1.go
Comment (BFostek) [inline _content/tour/por/variables/answer1.go:1]: As I discussed with Bill, we will not translate the code files; our focus will be on translating the articles

**Review by BFostek — DISMISSED** [review]

**Comment by ardan-bkennedy** [comment]
We should have a larger conversation but I think the code file comments should probably stay.

**Comment by rschio** [comment]
In welcome/hello.go and welcome/sandbox.go I changed the fmt.Println text, maybe I should revert this, what do you think?

**Comment by ardan-bkennedy** [comment]
Yea, I think that can be changed. It would be nice to see that in the right language.

**Comment by ardan-bkennedy** [comment]
@rschio you have been added to the branch and now have the same rights as everyone.

**Comment by rschio** [comment]
Ok, thank you!

**Comment by rschio** [comment]
I reverted the code translations, we agreed to only translate the articles.

File: _content/tour/por/welcome.article [hunk 1]
```diff
@@ -1,191 +1,192 @@
...
+O tour está dividido em uma lista de módulos que você pode
+accessar clicando em
+[[javascript:highlight(".logo")][Ultimate Go]] no canto superior esquerdo da página.
 
-You can also view the table of contents at any time by clicking on the [[javascript:highlightAndClick(".nav")][menu]] on the top right of the page.
+Você pode também ver a tabela de conteúdo a qualquer momento clicando no [[javascript:highlightAndClick(".nav")][menu]] no canto superior direito da página.
 
-Throughout the tour you will find a series of slides and exercises for you
-to complete.
+Ao longo do tour você vai encontrar uma série de slides e exercícios para você
```
[the remark this answers is not in the document]
Comment (luciormoraes) [reply _content/tour/por/welcome.article:16]: @rschio here.

File: _content/tour/por/variables.article [hunk 2]
```diff
@@ -65,8 +66,8 @@ performance.
     String             "" (empty)
     Pointer            nil
 
-** Declare and Initialize
-The keyword var can be used to construct values to their zero value state for all types.
+** Declarar e Inicializar
```
Comment (luciormoraes) [inline _content/tour/por/variables.article:69]: @rschio I'm wondering if we should use "Declaração e Inicialização". What do you think? But anyway I'm happy witht the current.

Comment (rschio) [reply _content/tour/por/variables.article:69]: Yeah I think "Declaração e Inicialização" fits better, I will fix.

**Review by BFostek — APPROVED** [review]

**Pull request merged**